__tests__/lib/prompts.test.ts

```diff
@@ -101,19 +79,11 @@ describe('prompt test bed', () => {
         )
       );
 
-      expect(answer.option).toBe('create');
+      expect(answer).toStrictEqual({ option: 'create' });
     });
 
     it('should return specId if user chooses to update file', async () => {
-      jest.mock('enquirer');
-      enquirer.on('prompt', async prompt => {
-        await prompt.keypress(null, { name: 'down' });
```

Comment:
We love to be forced to test a CLI UX like a React component don't we folks.

Reply:
such a goddamn mess